app.js

```diff
@@ -120,7 +119,7 @@ function checkQuadrantAndFinger(img, prediction, target) {
         return false
     }
     const quadrantCorrect = checkQuadrant(img, prediction.boundingBox, target.quadrant);
-    return checkFinger(img, prediction, target.numFingers).then(
+    return quadrantCorrect && checkFinger(img, prediction, target.numFingers).then(
         correct => {
             return correct && quadrantCorrect
         }
```

Comment:
no need to include `quadrantCorrect` again, and at that point it's just `correct => { return correct }`, which is just... nothing?

Reply:
can you test if it works without the entire `.then(...)` part?

Reply:
Whoa you're right it didn't break

Reply:
haha yayy, you fixed the problem in the other method, so this was fine to revert to what i originally had